### PR TITLE
Added improvements to account for autoscaled compute, and filter out non-master/infra/worker nodes.

### PR DESCRIPTION
**Pull request by aliceh**
Added code to account for autoscaled compute instances and filter for master/infra/worker in the instance name.

### CODE REVIEW

File: cmd/cluster/health.go [hunk 1]
```diff
@@ -164,7 +176,7 @@ func (o *healthOptions) run() error {
 //This command implements the ocm describe clsuter call via osm-sdk.
 //This call requires the ocm API Token https://cloud.redhat.com/openshift/token be available in the OCM_TOKEN env variable.
 //Example: export OCM_TOKEN=$(jq -r .refresh_token ~/.ocm.json)
-func ocmDescribe(clusterID string) ([]string, string, int, error) {
+func ocmDescribe(clusterID string) ([]string, string, int, int, int, int, int, error) {
```
Comment (iamkirkbater) [inline cmd/cluster/health.go:179]: I think with this many return parameters we should refactor this to be a specific struct, in order to make this more readable.

Maybe something like:

```
struct clusterDescribeOutput {
  az []string
  clusterName string
  compute int
  infra int
  master int
  autoscaleMin int
  autoscaleMax int
}
```
etc.

**Review by iamkirkbater — CHANGES_REQUESTED** [review]
I like the new feature, but it would be nice if it would be refactored for readability while the context is still fresh, please.

File: cmd/cluster/health.go (hunk 1 above)
Comment (georgettica) [reply cmd/cluster/health.go:179]: If you want some inspiration, see https://github.com/openshift/osdctl/pull/129/commits/4333c8aa3d2ab63634512fe93808dd7e530c4390

Comment (aliceh) [reply cmd/cluster/health.go:179]: Thanks!!

**Comment by aliceh** [comment]
I am so sorry, I somehow merged this from CLI by accident.

**Comment by aliceh** [comment]
https://github.com/openshift/osdctl/pull/137 this is the PR I wanted to submit. Sorry for accidentally merging something else into master.  @mmazur, @iamkirkbater could you please revert this?

**Pull request closed without merging**